src/apps/pipeline/client/AddToPipelineForm.jsx

```diff
@@ -0,0 +1,69 @@
+import React from 'react'
+import Button from '@govuk-react/button'
+import Link from '@govuk-react/link'
+import { FormStateful, FieldRadios, FormActions } from 'data-hub-components'
+import { connect } from 'react-redux'
+import Task from '../../../client/components/Task'
+
+import { PIPELINES__CHECKED_IF_ON_PIPELINE } from '../../../client/actions'
+import { state2props } from './state'
+import { companies, dashboard } from '../../../lib/urls'
+
+function saveCompanyToPipeline() {
+  return async () => {
+    // TODO: need to post tp the pipelines endpoint once it is built following the example below:
+    // await axios.patch(`/api-proxy/v4/company/${companyId}`, {
+    //   [EXPORT_WIN_FIELD_NAME]: status[EXPORT_WIN_FIELD_NAME] || null,
+    // })
+
+    return dashboard()
+    // TODO: I need to set an error message in the state and show the error here in this component if there was a problem with the post request
+  }
+}
+
+function AddToPipelineForm({ companyId, companyName, isOnPipeline }) {
+  return (
+    <Task.Status
+      name="Check if on pipeline"
+      id="checkIfOnPipeline"
+      progressMessage="checking pipeline..."
+      startOnRender={{
+        payload: { companyId },
+        onSuccessDispatch: PIPELINES__CHECKED_IF_ON_PIPELINE,
+      }}
+    >
+      {() =>
+        isOnPipeline ? (
+          <>
+            <p>{companyName} is already in your pipeline</p>
+            <p>
+              <a href={companies.detail(companyId)}>Go back to {companyName}</a>
+            </p>
+            <p>
+              <a href={dashboard()}>Go to your dashboard</a>
```

Comment:
I would suggest using the module name with urls so it's clear what are we using, as the `companies` var exists in many contexts, i.e.
```suggestion
              <a href={urls.dashboard()}>Go to your dashboard</a>
```

Reply:
FYI this code is going to be removed in a following PR with the suggested changes. in progress now. I think it safe to mark as resolved.